docker-compose.yml

```diff
@@ -15,35 +15,35 @@ services:
       timeout: 5s
       retries: 5
 
-  # subquery-node:
-  #   image: onfinality/subql-node:v0.33.0
-  #   depends_on:
-  #     'postgres':
-  #       condition: service_healthy
-  #   restart: always
-  #   environment:
-  #     DB_USER: postgres
-  #     DB_PASS: postgres
-  #     DB_DATABASE: postgres
-  #     DB_HOST: postgres
-  #     DB_PORT: 5432
-  #   volumes:
-  #     - ./:/app
-  #   command:
-  #     - -f=/app
-  #     - --db-schema=app
-  #     # - --unsafe
-  #     # - --force-clean
-  #   healthcheck:
-  #     test: ['CMD', 'curl', '-f', 'http://subquery-node:3000/ready']
-  #     interval: 3s
-  #     timeout: 5s
-  #     retries: 10
+  subquery-node:
+    image: onfinality/subql-node:v1.1.1
+    depends_on:
+      'postgres':
+        condition: service_healthy
+    restart: always
+    environment:
+      DB_USER: postgres
+      DB_PASS: postgres
+      DB_DATABASE: postgres
+      DB_HOST: postgres
+      DB_PORT: 5432
+    volumes:
+      - ./:/app
+    command:
+      - -f=/app
+      - --db-schema=app
+      - --unsafe
+      # - --force-clean
+    healthcheck:
+      test: ['CMD', 'curl', '-f', 'http://subquery-node:3000/ready']
+      interval: 3s
+      timeout: 5s
+      retries: 10
 
   # graphql-engine:
```

Comment:
You may want to uncomment the codes for subql query, I just comment for testing before